workflows/templates/drain-worker.yaml

```diff
@@ -65,6 +65,8 @@ spec:
                 value: "{{inputs.parameters.dryRun}}"
               - name: scriptContent
                 value: |
+                  echo "Sleeping for 60 seconds to allow pod migration to commence"
+                  sleep 60
```

Comment:
Do not check pods immediately after drain - it may need a few seconds for pods to start terminating.